go.mod

```diff
@@ -32,7 +33,9 @@ require (
 
 require (
 	github.com/Azure/go-ntlmssp v0.0.0-20221128193559-754e69321358 // indirect
+	github.com/BurntSushi/toml v1.3.2 // indirect
```

Comment:
@jjngx @pdabelf5 do we need to get approval for new libs?

Reply:
The only direct dependency added is `github.com/gruntwork-io/terratest v0.47.2`

Reply:
@vepatel both dependencies have MIT license. It should be straightforward.

Reply:
Thanks @jjngx

Reply:
`github.com/gruntwork-io/terratest` has a pre-approved Apache 2.0 license